### PR TITLE
add assertion on NSTemplateTier.spec.clusterResource

### PR DESCRIPTION
**Pull request by xcoulon**
associated with https://github.com/codeready-toolchain/host-operator/pull/157

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -116,8 +116,12 @@ github.com/codeready-toolchain/api v0.0.0-20200226022556-bb2d9dd17828 h1:lWjwDBd
 github.com/codeready-toolchain/api v0.0.0-20200226022556-bb2d9dd17828/go.mod h1:w07CIyd2cCp1i38hdPrG4aZPbwISzMkxOmtCWE4EtdY=
 github.com/codeready-toolchain/api v0.0.0-20200226154004-addd941b675c h1://VqLgLGvtUiWcJn3qju6PjIoc01O+QrVD19oK/s8Zk=
 github.com/codeready-toolchain/api v0.0.0-20200226154004-addd941b675c/go.mod h1:w07CIyd2cCp1i38hdPrG4aZPbwISzMkxOmtCWE4EtdY=
+github.com/codeready-toolchain/api v0.0.0-20200305152414-a958daa81263 h1:LyzNCGqUwk6j9q48vFdLriHPEyGsZcQcvOtq/ZbHA6w=
```
Comment (alexeykazakov) [inline go.sum]: you could use `go mod tidy` to remove unused dependencies.

Comment (xcoulon) [reply go.sum]: done in 32664fb

**Comment by alexeykazakov** [comment]
The test should be fixed by https://github.com/codeready-toolchain/host-operator/pull/157/commits/28eaba6064376ab4ef7fedf73d90379018881fb6

**Comment by alexeykazakov** [comment]
This PR needs to be re-created to be paired with https://github.com/codeready-toolchain/host-operator/pull/161

**Comment by xcoulon** [comment]
replaced by https://github.com/codeready-toolchain/toolchain-e2e/pull/81

**Comment by xcoulon** [comment]
replaced by https://github.com/codeready-toolchain/toolchain-e2e/pull/81

**Pull request closed without merging**